pkg/controller/servicebindingrequest/mapper_test.go

```diff
@@ -4,44 +4,142 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"github.com/redhat-developer/service-binding-operator/pkg/testutils"
+	"github.com/redhat-developer/service-binding-operator/test/mocks"
 )
 
 func TestSBRRequestMapperMap(t *testing.T) {
-	mapper := &sbrRequestMapper{}
+	sbr := &v1alpha1.ServiceBindingRequest{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ServiceBindingRequest",
+			APIVersion: "apps.openshift.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "mapper-unit",
+			Name:      "mapper-unit-sbr",
+		},
+		Spec: v1alpha1.ServiceBindingRequestSpec{
+			ApplicationSelector: v1alpha1.ApplicationSelector{
+				GroupVersionResource: metav1.GroupVersionResource{
+					Group:    "apps",
+					Version:  "v1",
+					Resource: "deployments",
+				},
+				ResourceRef: "mapper-unit-deployment",
+			},
+			BackingServiceSelector: &v1alpha1.BackingServiceSelector{
```

Comment:
since, we plan to deprecate `v1alpha1.BackingServiceSelector`, can we use `v1alpha1.BackingServiceSelectors` over here?

Reply:
Solved in bc678c3e3dcce1851d3ec5a006e9bd6d170f5c43.